### PR TITLE
Correct RegisterListenersPass namespace

### PR DESCRIPTION
**Pull request by hacfi**
See https://github.com/symfony/symfony/pull/9792

### CODE REVIEW

File: components/event_dispatcher/introduction.rst [hunk 1]
```diff
@@ -212,14 +212,14 @@ instance of ``Symfony\Component\HttpKernel\Event\FilterResponseEvent``::
     and the
     :doc:`DependencyInjection component </components/dependency_injection/introduction>`,
     you can use the
-    :class:`Symfony\\Component\\HttpKernel\\DependencyInjection\\RegisterListenersPass`
-    from the HttpKernel component to tag services as event listeners::
+    :class:`Symfony\\Component\\EventDispatcher\\DependencyInjection\\RegisterListenersPass`
+    from the EventDispatcher component to tag services as event listeners::
```
Comment (xabbuh) [inline components/event_dispatcher/introduction.rst]: I think we can now remove the "from the EventDispatcher component" part (we are talking about the EventDispatcher component here).

Comment (hacfi) [inline components/event_dispatcher/introduction.rst]: @xabbuh Thanks..I was really tired when I created the PR :).

**Comment by xabbuh** [comment]
Thanks @hacfi :+1: 

@weaverryan @WouterJ This should be merged into the `2.6` branch.

**Comment by weaverryan** [comment]
Ah, very good catch! Merge into 2.6 - thanks @hacfi!

**Pull request closed without merging**